packages/app/src/cli/utilities/developer-platform-client/app-management-client.ts

```diff
@@ -280,8 +289,16 @@ export class AppManagementClient implements DeveloperPlatformClient {
       )
   }
 
-  async templateSpecifications(_appId: string): Promise<ExtensionTemplate[]> {
-    return stubbedExtensionTemplates()
+  async templateSpecifications({organizationId}: MinimalAppIdentifiers): Promise<ExtensionTemplate[]> {
```

Comment:
Would like to see a test before moving out of draft.

Reply:
I believe I've tested this reasonably thoroughly, let me know if something is missing.